src/main/java/callhub/connect/drivers/MessageWebSocketAccess.java

```diff
@@ -0,0 +1,17 @@
+package callhub.connect.drivers;
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.messaging.handler.annotation.SendTo;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class MessageWebSocketAccess {
+
+    @MessageMapping("/message")
+    @SendTo("/topic/messages")
+    public String sendMessage(String message) throws Exception {
+        Thread.sleep(1000); // simulated delay
```

Comment:
Do we need this simulated delay? is there a way to wrap this in a method so it is not visible on the controller code?

Reply:
Removed the delay